fullstop-apis/fullstop-violation-api/src/main/java/org/zalando/stups/fullstop/violation/Violation.java

```diff
@@ -35,5 +35,9 @@
 
     String getViolationType();
 
+    String getUsername();
+
+    Boolean getChecked();
```

Comment:
the boolean to resolve the violation, but I think is now dynamic and depends on message presents. Will ask if we can delete it! thx